src/pages/LoginScanStep.tsx

```diff
@@ -1,14 +1,51 @@
-import { Box, Typography, Button, Link } from '@mui/material'
-import { SVGLogoDescreption, SVGALoginLogo, SVGQRCode } from '../assets/svgs'
-import { useNavigate } from 'react-router-dom'
-import { login } from '../tools/auth'
+import { Box, Typography, CircularProgress } from '@mui/material'
+import { SVGLogoDescreption, SVGALoginLogo } from '../assets/svgs'
+import { useState, useEffect } from 'react'
+import QRCode from 'react-qr-code'
+import ErrorIcon from '@mui/icons-material/Error'
+import { v4 as uuidv4 } from 'uuid'
+import { getOrCreateAppInstanceDid } from '@cooperation/vc-storage'
 
 export default function LoginScanStep() {
-  const navigate = useNavigate()
+  const [qrData, setQrData] = useState('')
+  const [isLoading, setIsLoading] = useState(true)
+  const [error, setError] = useState('')
 
-  const handleGoogleSignIn = () => {
-    login('/resume/import')
-  }
+  useEffect(() => {
+    (async () => {
+      const sessionId = uuidv4()
+      const { did: resumeDid } = await getOrCreateAppInstanceDid()
+
+      const SERVER_URL = 'http://192.168.1.26:3000'
+      const WALLET_DEEP_LINK = 'walletapp://import'
```

Comment:
```suggestion
      const WALLET_DEEP_LINK = ''https://lcw.app/request''
```

The deep link needs to be `'https://lcw.app/request'` (that's what opens LCW automatically, if you scan it with the phone camera)